ImagineAR/IAR-SDK-Sample-iOS
# IAR-5361 Fix distance label [Surface]

### Overview

- Add Location command as a custom command for debug tools
- Fix location distance for Surface

### Resources

IAR-5361

### Testing Instructions

Select "Take me here" on any location marker and see the distance label marking 0
